Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM buildpack-deps:stretch-curl
+FROM buildpack-deps:bullseye-curl
```

Comment:
can you please update the `echo` on line 5 as well

Reply:
Updated